tardis_env3.yml

```diff
@@ -8,17 +8,17 @@ dependencies:
   # WARNING: any change to this section must be applied to the conda-forge
   # package recipe at https://github.com/conda-forge/tardis-sn-feedstock
 
-  - python=3.8
+  - python =3.8
   - setuptools
-  - setuptools_scm<7
+  - setuptools_scm <7
```

Comment:
If you're changing the formatting, why not spaces on both sides of the operator?

Reply:
It's the conda-forge standard, I thought would be a good idea to keep the same format.